viz/metrics-api/prometheus.go

```diff
@@ -118,8 +118,14 @@ func promDstGroupByLabelNames(resource *pb.Resource) model.LabelNames {
 func promQueryLabels(resource *pb.Resource) model.LabelSet {
 	set := model.LabelSet{}
 	if resource != nil {
-		if resource.Name != "" && resource.GetType() != k8s.Service {
-			set[promResourceType(resource)] = model.LabelValue(resource.Name)
+		if resource.Name != "" {
+			if resource.GetType() == k8s.Server {
```

Comment:
Is this change required? In `stat_summary.go`, We use the resource labels directly? We can keep this change and remove the manual setting there? :thinking:

Reply:
This was to fix a bug when calling stat summary with a server or saz as the ToResource.  In this case it wasn't rewriting the label name and was attempting to query prometheus with `dst_server=` instead of `srv=` or something like that.